src/lib/minions/functions/removeFoodFromUser.ts

```diff
@@ -38,6 +38,13 @@ export default async function removeFoodFromUser({
 		totalHealingNeeded = reduceNumByPercent(totalHealingNeeded, 17.5);
 		reductions.push(`-17.5% for Ely ${Emoji.Ely}`);
 	}
+	if (
+		gearSetupsUsed.some(i => i[0] === 'melee') &&
+		rawGear.melee.hasEquipped(['Justiciar faceguard', 'Justiciar chestguard', 'Justiciar legguards'], true, true)
+	) {
+		totalHealingNeeded = reduceNumByPercent(totalHealingNeeded, 12.5);
+		reductions.push('-6% for Justiciar');
```

Comment:
Reduced percent doesn't match boost message. Definitely think 12.5% is better, in game in can easily soak that much or more damage.

Shouldn't be too hard to make it 'accurate' also, just need:
```ts
const gearStat = ${total stats of combat style were defending against};
const justiciarReductionPercent = (gearStat / 3000) * 100;
totalHealingNeeded = reduceNumByPercent(totalHealingNeeded, justiciarReductionPercent);
reductions.push(`${justiciarReductionPercent.toFixed(1)}% for Justiciar`);
```